medusa/storage/google_storage.py

```diff
@@ -151,7 +154,7 @@ async def _download_blob(self, src: str, dest: str):
             stream = await self.gcs_storage.download_stream(
                 bucket=self.bucket_name,
                 object_name=object_key,
-                timeout=-1,
+                timeout=self.read_timeout if self.read_timeout is not None else -1,
```

Comment:
Hello @rzvoncek,

I am a user of the k8ssandra operator on gcp and this change can make a breaking change in the restore case if the files are big. We are switching from infinite to 60 seconds timeout if I am not mistaken.

Do you think we should rollback the change for the gcp case or should I post a comment here https://github.com/k8ssandra/k8ssandra-operator/issues/1353 to know if we know when this will be work on ?